dxr/plugins/rust/tests/test_data/test_fields.py

```diff
@@ -52,29 +52,29 @@ def test_qual_enum_field_ref(self):
         self.found_line_eq('+var-ref:test::EnumFields::Struct::field1', "EnumFields::Struct{ <b>field1</b>, field2 } =&gt; {}", 42)
         self.found_line_eq('+var-ref:test::EnumFields::Struct::field2', "EnumFields::Struct{ field1, <b>field2</b> } =&gt; {}", 42)
 
-    def test_field_ref(self):
```

Comment:
So, did we break some functionality, or what? If it does turn out to be a good idea to temporarily disable this test, we should `raise SkipTest('The reason for skipping them')` as the first line of the test instead of commenting them out. That way, we have the reason recorded, and we have the guilt of a high "skipped" count at the end of the test run to motivate us.